flake.nix

```diff
@@ -17,37 +16,39 @@
   outputs = {
     self,
     nixpkgs,
-    flake-utils,
     rust-overlay,
     ...
-  }:
-    {
-      overlays.default = final: prev: {inherit (self.packages.${prev.system}) swww;};
-    }
-    // flake-utils.lib.eachDefaultSystem (
-      system: let
-        inherit (nixpkgs) lib;
-
-        pkgs = import nixpkgs {
-          inherit system;
-          overlays = [(import rust-overlay)];
-        };
-
-        cargo-toml = lib.importTOML ./Cargo.toml;
-        inherit (cargo-toml.workspace.package) rust-version;
-        rust = pkgs.rust-bin.stable.${rust-version}.default;
-
-        rustPlatform = pkgs.makeRustPlatform {
-          cargo = rust;
-          rustc = rust;
-        };
-      in {
-        packages = {
-          swww = rustPlatform.buildRustPackage {
+  }: let
+    inherit (nixpkgs) lib;
+    systems = [
+      "x86_64-linux"
+      "aarch64-linux"
+      "x86_64-darwin"
+      "aarch64-darwin"
+    ];
```

Comment:
Why are you specifying systems like this? I've gotten issues from people who use `swww` in `x86` systems (not `x86_64`. I don't see what specifying things like this does other than arbitrarily limiting the program's usability.